Makefile

```diff
@@ -175,9 +188,9 @@ migrate:  ## run django migrations for the joanie project.
 	@$(MANAGE) migrate
 .PHONY: migrate
 
-superuser: ## create a Django superuser
+superuser: ## Create an admin superuser with password "admin"
 	@echo "$(BOLD)Creating a Django superuser$(RESET)"
-	@$(MANAGE) createsuperuser
+	@$(MANAGE) shell -c "from joanie.core.models import User; not User.objects.filter(username='admin').exists() and User.objects.create_superuser('admin', 'admin@example.com', 'admin')"
```

Comment:
I realized we are to use createsuperuser just by setting the environment variable `DJANGO_SUPERUSER_PASSWORD=admin`.

Then use this command : 
```Makefile
@$(MANAGE) createsuperuser --username admin --email admin@example.com --no-input
```

Reply:
Ok I modified but it's not much better :man_shrugging: 
```
@$(COMPOSE_RUN) -e DJANGO_SUPERUSER_PASSWORD=admin app-dev python manage.py createsuperuser --username admin --email admin@example.com --no-input
```